### PR TITLE
Feature/issue 79 - Generate performance benchmarking test data

### PR DESCRIPTION
**Pull request by torimcd**
Github Issue: #79 

### Description

Add many time steps for a test reach id to the database to enable performance testing.

### Overview of work done

Added benchmarking_data_reaches.csv file containing 1200 timesteps for reach_id "00001100011". This is a fake reach id which does not exist in the SWORD database and so should not interfere with the real data. Note, the leading zeros are required.

Added new module to io/swot_reach_node_shp.py to unpack the CSV file. 
Added new module to load_data script to send this data to dynamo db. 
Added new flag in load_granule lambda handler to determine when test data should be loaded. 
Added batch writer to the db schema to pass handling batch writes and retries to boto3.
Updated load data readme documentation to describe how to run in AWS console.

### Overview of verification done

No new unit tests added, existing tests continue to pass. 

### Overview of integration done

Deployed to SIT and successfully loaded 1200 items for reach_id=00001100011 into db.

## PR checklist:

* [x] Linted
* [x] Updated unit tests
* [x] Updated changelog
* [x] Integration testing

_See [Pull Request Review Checklist](../CONTRIBUTING.md#reviewing) for pointers on reviewing this pull request_

### CODE REVIEW

**Review by nikki-t — APPROVED** [review]
This looks good to me. I like the use of batch writes to DynamoDB to optimize writes to the database and that the AWS resource handles batching the items written to the table.

File: hydrocron/db/load_data.py [hunk 1]
```diff
@@ -69,11 +73,21 @@ def granule_handler(event, _):
...
+        s3_resource = connection.s3_resource
 
-    s3_resource = connection.s3_resource
-    items = read_data(granule_path, obscure_data, s3_resource)
+        logging.info("Starting read granule")
+        items = read_data(granule_path, obscure_data, s3_resource)
 
+    print("Set up dynamo connection")
     dynamo_resource = connection.dynamodb_resource
+    print("loading data items")
```
Comment (frankinspace) [inline hydrocron/db/load_data.py:90]: Can you switch these to `logging` statements or remove?

File: hydrocron/db/load_data.py [hunk 2]
```diff
@@ -152,6 +168,7 @@ def load_data(dynamo_resource, table_name, items):
     """
 
     try:
+        print("Set dynamo table connection")
```
Comment (frankinspace) [inline hydrocron/db/load_data.py:171]: Same, use `logging`

Comment (torimcd) [reply hydrocron/db/load_data.py]: done

**Pull request merged**